ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -590,6 +590,13 @@ pub(crate) struct BBlock {
     pub(crate) instrs: TiVec<InstrIdx, Instruction>,
 }
 
+impl BBlock {
+    // Returns true if this block contains a return, false otherwise.
```

Comment:
Should be `///` (not `//`).

Also I think it would be better to say "if this block is terminated by a return"?